galsim/fits.py

```diff
@@ -537,6 +540,10 @@ def write(image, file_name=None, dir=None, hdu_list=None, clobber=True, compress
     """
     from galsim._pyfits import pyfits
 
+    if image.iscomplex:
```

Comment:
The use case I had in mind was debugging, mostly, for cases where I sometimes want to see what something looks like in Fourier space.  But anyway I'm fine with having this as a separate issue.